README.md

```diff
@@ -1,70 +1,198 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
+  <!-- <img src="./app_screenshot.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
 
-## Available Scripts
+  <h3><b>Space Travelers' Hub</b></h3>
 
-In the project directory, you can run:
+</div>
 
-### `npm start`
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Space Travelers' Hub ](#-space-travelers-hub-)
+  - [Screenshot ](#screenshot-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+      - [`npm start`](#npm-start)
+    - [Run tests](#run-tests)
+      - [`npm test`](#npm-test)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🔭 Future Features ](#-future-features-)
+  - [Acknowledgements ](#acknowledgements-)
+  - [📝 License ](#-license-)
+
+# 📖 Space Travelers' Hub <a name="about-project"></a>
+
+**Space Travelers' Hub** is a simple app that fetches data from SpaceX API and provide the ability to reserve rockets and join missions. Built with <a href="https://reactjs.org/">React.js</a>
+
+## Screenshot <a name="screenshot"></a>
+
+<div align='center'>
+  <img src="./public/screenshot.png" alt="logo" width="400"  height="auto" />
+    <br/>
+</div>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Single Page Application]**
+- **[Reusable Components]**
+- **[Client side routing]**
+- **[Use remote data from API]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+https://space-travelers-hub-qfz3.onrender.com/
```

Comment:
- [OPTIONAL] It will be nice to remove this url as the will make your readme file cleaner as the demo link already exist below.